src/lib/Microsoft.Health.Fhir.Ingest.Template/Microsoft.Health.Fhir.Ingest.Template.csproj

```diff
@@ -1,10 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I've gone ahead and updated this to `netcoreapp3.1`. This was needed in order to use the `Microsoft.Health.Logging` package. This hasn't caused any adverse affects AFAIK but please let me know if this could cause issues.